packages/plugins/other/visitor-plugin-common/src/client-side-base-visitor.ts

```diff
@@ -228,8 +230,14 @@ export interface ClientSideBasePluginConfig extends ParsedConfig {
   pureMagicComment?: boolean;
   optimizeDocumentNode: boolean;
   experimentalFragmentVariables?: boolean;
+  unstable_onDocumentNode?: Unstable_OnDocumentNode;
+  unstable_omitDefinitions?: boolean;
```

Comment:
Added them as unstable as I don't want anyone outside the client preset to use these features tbh.